test/topology/test_null.py

```diff
@@ -5,23 +5,18 @@
 #
 from cassandra.protocol import InvalidRequest   # type: ignore
 from pylib.util import unique_name
+from pylib.schema_helper import get_schema
 import pytest
 
 
-@pytest.fixture(scope="session")
-async def table1(cql, keyspace):
-    table = keyspace + "." + unique_name()
-    cql.execute(f"CREATE TABLE {table} (p text, c text, v text, primary key (p, c))")
-    yield table
-    await cql.run_async("DROP TABLE " + table)
-
-
 @pytest.mark.asyncio
-async def test_delete_empty_string_key(cql, table1):
+async def test_delete_empty_string_key(cql):
+    tables = await get_schema("delete_empty_string_key", cql, ntables=1, ncolumns=5)
     s = "foobar"
     # An empty-string clustering *is* allowed:
-    await cql.run_async(f"DELETE FROM {table1} WHERE p='{s}' AND c=''")
+    await cql.run_async(f"DELETE FROM {tables[0]} WHERE pk = '{s}' AND {tables[0].columns[1].name} = ''")
```

Comment:
One of the things I don't like about large utilities like that "schema helper" is that it obscures what this test is really testing:
1. Does the table returned by get_schema always have a string partition key? Why is this the case, and how does the reader of this test code guess this?
2. Why does this test have 5 columns, and what's the point of them being random?

Reply:
test_null removed in v8

PK made random type in v8